hack/generated/pkg/testcommon/kube_per_test_context.go

```diff
@@ -191,7 +197,7 @@ func (tc KubePerTestContext) CreateNewTestResourceGroup(wait WaitCondition) (*re
 		// doesn't stop polling resources in "Deleting" state and so when running recordings
 		// different runs end up polling different numbers of times. Ensuring we reach a state
 		// the controller deems terminal (Deleted) resolves this issue.
-		tc.G.Eventually(rg, 2*time.Minute).Should(tc.Match.BeDeleted())
+		tc.G.Eventually(rg, 5*time.Minute).Should(tc.Match.BeDeleted())
```

Comment:
Perhaps should be in a set of “test constants” somewhere

Reply:
Yeah this is awkward because in replay mode it doesn't really need to be this long but in record mode I've seen a few tests fail because it was too short. Not consistently though. I figured bumping it up was _probably_ ok, but can pull it out into a constant.

Reply:
Pulled this up into a `ResourceGroupDeletionWaitTime` at the top of this file for now